lib/node/node.ts

```diff
@@ -52,3 +52,21 @@ if (crypto) {
     }
   }.bind(crypto);
 }
+
+// HTTP Client
+let httpClient;
+try {
+  httpClient = require('_http_client');
+} catch (err) {}
+
+if (httpClient && httpClient.ClientRequest) {
+  let ClientRequest = httpClient.ClientRequest.bind(httpClient);
+  httpClient.ClientRequest = function(options: any, callback?: Function) {
+    if (!callback) {
+      return new ClientRequest(options);
+    } else {
+      let zone = Zone.current;
+      return new ClientRequest(options, zone.wrap(callback, '_http_client.ClientRequest.constructor'));
```

Comment:
why `_http_client.ClientRequest.constructor`? Why not just `http.ClientRequest`?